src/pybind/mgr/cephadm/module.py

```diff
@@ -1495,6 +1494,9 @@ def _create_daemon(self, daemon_type, daemon_id, host,
             extra_config = {}
         name = '%s.%s' % (daemon_type, daemon_id)
 
+        if redeploy:
+            extra_args.extend(['--skip-port-check'])
```

Comment:
hm. wondering. should we instead add a `--redeploy` option?

Reply:
instead of adding yet another flag, maybe we just check the unit status (#35114) ?

https://github.com/ceph/ceph/pull/35114/commits/c85a34aaa70aca55a79e11396b57b31ded14d0b3#diff-8b586ec9c3ad3e8421a8858888f7ddf0R2684-R2688

Reply:
+1 for not adding a new flag, if we don't need it